x/stakeibc/keeper/unbonding.go

```diff
@@ -393,14 +372,13 @@ func (k Keeper) UnbondFromHostZone(ctx sdk.Context, hostZone types.HostZone) (er
 		return err
 	}
 
-	// Sum the total number of native tokens that from the records above that are ready to unbond
-	totalNativeUnbondAmount, totalStBurnAmount := k.GetTotalUnbondAmount(epochNumberToHostZoneUnbonding)
+	// Sum the total number of native tokens from the records above that are ready to unbond
+	totalUnbondAmount := k.GetTotalUnbondAmount(epochNumberToHostZoneUnbonding)
```

Comment:
The name epochNumberToHostZoneUnbonding confused me because it sounded singular -- could we do epochNumbersToHostZoneUnbondings for clarity?

Reply:
Still could be useful to call it totalNativeUnbondAmount just for clarity IMO

Reply:
good call, updated